client/js/components/user.jsx

```diff
@@ -0,0 +1,74 @@
+var React = require("react");
+var c3 = require("c3");
+
+var User = React.createClass({
+  componentDidMount: function() {
+    var chart = c3.generate({
+      bindto: ".contribution-grade",
+      data: {
+        columns: [["data", 0]],
+        type: "gauge",
+        onclick: function(d, i) {
+          console.log("onclick", d, i);
+        },
+        onmouseover: function(d, i) {
+          console.log("onmouseover", d, i);
+        },
+        onmouseout: function(d, i) {
+          console.log("onmouseout", d, i);
+        }
+      },
+      gauge: {
+        label: {
+          format: function(value, ratio) {
+            return value;
+          },
+          show: true
+        },
+        min: 0,
+        max: 6,
+        units: "schemas",
+        width: 39
+      },
+      color: {
+        pattern: ["#FF0000", "#F97600", "#F6C600", "#60B044"],
+        threshold: {
+          unit: "value"
+        }
+      },
+      size: {
+        height: 150
+      }
+    });
+
+    var CHART_LOAD_TIME = 500;
+    setTimeout(function () {
+      chart.load({
+        columns: [["data", 4]]
+      });
+    }, CHART_LOAD_TIME);
+  },
+  render: function() {
+    return (
+      <div className="my-data-page">
+        <section className="contribution">
+          <h2 className="user-welcome">Welcome back!</h2>
+          <p>Here's your contribution grade for the week so far:</p>
```

Comment:
I wonder if the "'" should be escaped, or if the github diff is just freaking me out...